apps/rig_inbound_gateway/lib/rig_inbound_gateway/api_proxy/handler/http.ex

```diff
@@ -43,6 +45,26 @@ defmodule RigInboundGateway.ApiProxy.Handler.Http do
           "Failed to proxy to #{inspect(url)} (#{method} #{request_path}): #{inspect(err)}"
         end)
 
+        case err.reason do
```

Comment:
could you please move this to a dedicated function? For example:

```elixir
count_http_request_error(conn, err)
```

With the implementation

```elixir
defp count_http_request_error(conn, %HTTPoison.Error{reason: reason}) do
  ...
end
defp count_http_request_error(conn, :timeout) do
  ...
end
# any other errors are not counted:
defp count_http_request_error(_, _), do: :ok
```